.nxignore

```diff
@@ -1,5 +1,6 @@
-# helm
-/helm
+# helm related files
+apps/**/infra/*
+libs/**/infra/*
```

Comment:
Isn't the infra/ and charts/ from the root missing?

Reply:
I think they are not included as is since nothing no project is referring to those folders in workspace.json but I might be missing something.